modules/menu/static_menu.c

```diff
@@ -673,18 +699,15 @@ static int cmd_ua_next(struct re_printf *pf, void *unused)
 	(void)pf;
 	(void)unused;
 
-	if (!menu->le_cur)
```

Comment:
we should consider to remove this command (toggle ua / ua_next).

it would be nice if there is no "current" ua